devito/arch/compiler.py

```diff
@@ -373,7 +373,7 @@ def add_ldflags(self, flags):
 class GNUCompiler(Compiler):
 
     def __init__(self, *args, **kwargs):
-        super(GNUCompiler, self).__init__(*args, **kwargs)
```

Comment:
I don't understand why this line has changed for graviton support.

Reply:
Oh it's just we moving towards `super()` everywhere in the codebase rather than with args so changed it while there